### PR TITLE
Fix config console.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

**Comment by terabytesoftw** [comment]
Before merge: https://github.com/yiisoft/yii-event/pull/9

File: config/providers-web.php [hunk 1]
```diff
@@ -3,14 +3,7 @@
...
-use Yiisoft\Composer\Config\Builder;
-use Yiisoft\Yii\Event\EventDispatcherProvider;
 
 return [
-    'yiisoft/event-dispatcher/eventdispatcher' => [
-        '__class' => EventDispatcherProvider::class,
-        '__construct()' => [Builder::require('events-web')]
-    ],
-
     ReverseBlockMerge::class => new ReverseBlockMerge(),
```
Comment (viktorprogger) [inline config/providers-web.php:8]: Is it necessary?

**Review by viktorprogger — CHANGES_REQUESTED** [review]

**Comment by viktorprogger** [comment]
There are some more config arrays with an only `ReverseBlockMerge` element. Can we remove that element in this PR?

File: config/providers-web.php (hunk 1 above)
Comment (samdark) [reply config/providers-web.php:8]: Yes.

**Review by samdark — APPROVED** [review]

**Pull request merged**